tests/ui_tests/fixtures.json

```diff
@@ -1,4 +1,12 @@
 {
+"cardano-test_sign_tx.py::test_cardano_sign_tx[parameters0-result0]": "b1f022d81fb324a136a8401232b7112bca2b5212b98578a189d3486911d469cc",
```

Comment:
`parameters0-result0` is a bit less clearer then `[m-44'-1815'-0'-0-0-Ae2tdPwUPEZLCq3sFv4wVYx": "5a80508a71a9ef64f94762b07636f90e464832f0f4a3102af8fa1a8c69e94586]` I guess, are we okay with this? Maybe we might include the test description somehow or introduce "ids".

Reply:
personally I don't think the latter is any clearer :)

i'm OK with this (and both 0s are sequence numbers i think?), and if we need clarity, we can introduce ids easily: `pytest.param` has an `id` parameter that is easy to take from the json

Reply:
Ok let's leave it as it is for now then?